tox.ini

```
@@ -110,6 +110,21 @@ commands =
         --until Osaka \
         tests
 
+[testenv:benchmark]
+description = Fill the benchmark tests using EELS (with Python)
+commands =
+    fill \
+        --generate-all-formats \
+        --gas-benchmark-values 5 \
```

If its just sanity I think 1 will be faster?
```suggestion
        --gas-benchmark-values 1 \
```